mscore/albummanager.h

```diff
@@ -32,8 +32,8 @@ struct AlbumScore : public QObject {
 
 public:
     MasterScore* score                  { nullptr };
-    QTableWidgetItem* listItem          { nullptr };
-    QTableWidgetItem* listDurationItem  { nullptr };
+    QTableWidgetItem* listItem          { nullptr }; // αυτά δεν πρεπει να ειναι εδω
+    QTableWidgetItem* listDurationItem  { nullptr }; // και αυτο
```

Comment:
While I can understand that this a) is Work In Progress and b) Greek is your native language and as such more comfortable to you, I'd still prefer comments in English, as soon as it pets published here on GitHub

Reply:
I will take care of that tomorrow :-)

Reply:
Done